app/views/admin/students/_form.html.erb

```diff
@@ -61,6 +61,7 @@ and renders all form fields for a resource's editable attributes.
     <%= f.submit %>
   </div>
 <% end %>
+<% if page.resource.persisted? %>
```

Comment:
That's a good idea! Instead of checking different action, you can check persisted 👍